### DISCUSSION TITLE
Connection info system spam

### DISCUSSION THREAD

**Discussion by crispinknox** [azerothcore/azerothcore-wotlk#19374] [General]
Hi I recently resetup azerothcore but I am getting spammed in system chat with: 

Mean: xxms
Median: xxms
Percentiles: (95, 99, max) xxms, xxms, xxms

(xx being a two digit number)

I do not remember this being the case the last time i used azerothcore.

Can anyone advise a setting to change so this no longer happens?

**Comment by Dimi-Dun-Morogh** [comment]
any info how to turn this off?

**Reply by maldiablo** [reply]
I found the spam comes from the addons AzerothAdmin or AzerothCoreAdmin

AzerothCore changed what is output when you run the '.server info' command.  The addons run this command regularly but no longer know what to do with the new output so they dump it to chat.

My workaround was to go into the addons lua files and find any text with self.ChatMsg(".server info") and comment those lines out.  The creators of those addons need to update to fix the issue properly.

**Reply by GyllenByte** [reply]
The fact that the UI has no off toggle for the chat spam is irritating.

Commented this out in AzerothCoreAdmin.lua (lines 233+)

```lua
function MangAdmin:OnEnable()
  self:SetDebugging(true) -- to have debugging through the whole app
  ma_toptext:SetText(Locale["label_infoChar"].." "..Locale["label_infoGUID"]..tonumber(UnitGUID("player"),16))
  ma_top2text:SetText(Locale["label_infoRealm"])
  self:SearchReset()
  -- refresh server information
  -- self:ChatMsg(".server info")
  -- self:ChatMsg(".account onlinelist")
```

I think this did the trick. I play solo. The constant chat spam was driving me nuts. Esp. inbetween tweaking ollama bot chatter.
